scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -307,9 +307,15 @@ function Initialize-IoTEdge {
         throw
     }
 
+    if (-not (Test-VcRuntimePresent)) {
+        Write-HostRed 'VC Runtime must be installed before IoT Edge can be initialized.'
+        throw
+    }
+
+
     if ((Test-MobyNeedsToBeMoved) -or (Test-LegacyInstaller)) {
         Write-HostRed
-        Write-HostRed ('IoT Edge is installed in an invalid location. ' + $ReinstallMessage)
+        Write-HostRed ('IoT Edge or the IoT Edge Moby Engine is installed in an invalid location. Probably there is an old preview install present and it needs to be uninstalled or the system should be re-imaged.' + $ReinstallMessage)
```

Comment:
`There may be an old preview install of IoT Edge still present. Please run Uninstall-IoTEdge first or re-image the device.`

Reply:
fixed